content/5_Plat/Sweep_Line.problems.json

```diff
@@ -174,7 +174,8 @@
       "isStarred": false,
       "tags": ["1DRQ", "Sweep Line"],
       "solutionMetadata": {
-        "kind": "none"
+        "kind": "link",
+        "url": "https://usaco.guide/plat/range-sweep?lang=cpp#solution---intersection-points"
```

Comment:
ok since this problem is a focus problem in a module i think you should jsut delete it from the json completely

Reply:
okay